Merge new FontColorPicker into Master.

IsabelKoe opened the pull request:
Hey, I am finished with the implementation of the new FontColorPicker Logic. We have to think about deleting the old component.
Please also provide feedback with regards to the styling of the Text Sidebar.

Thank you, Isabel

ON21MB reviewed (approved):
For me that's perfect!

IsabelKoe commented:
Hey, I adjusted the coloroicker logic a bit and implemented the same layout and logic to the background color picker. After that I merged the current master into the branch. Could you please check if we can merge it into the master? Thanks, Isabel

@ON21MB @ON21-RK

ON21-RK reviewed (approved):
Look's great! :)
